### ISSUE TITLE
Add support for Boo

### ISSUE THREAD

**Issue by Ragrox123** [atom-ide-community/atom-script#291]
Please, add support for Boo is my favorite language.

**Comment by rgbkrk** [comment]
Things we'll need:

* What's the language package for Boo?
* How do you run a boo script?

**Comment by Ragrox123** [comment]
package name is "Boo-Language"To run a script you use the command "booi scriptname.boo"
Thanks hope this helps 

Date: Tue, 17 Feb 2015 08:35:01 -0800

**Comment by erran** [comment]
This is actually a duplicate of #173. That issue hasn't seen much activity though.

1. One more question: Can we run a code selection with `booi` or will there only be file based runs?

**Comment by Ragrox123** [comment]
No, only with the file directory, but I can compile to run a code selection
Date: Tue, 17 Feb 2015 19:28:00 -0800
